learning/katas/go/Core Transforms/Combine/CombineFn/pkg/task/task.go

```diff
@@ -0,0 +1,55 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"github.com/apache/beam/sdks/go/pkg/beam"
+)
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
+	return beam.Combine(s, &averageFn{}, input)
+}
+
+type averageAccum struct {
+	Count int64
+	Sum float64
+}
+
+type averageFn struct{}
+
+func (c *averageFn) CreateAccumulator() averageAccum {
```

Comment:
Is there any good documentation that outlines what function names that need to be implemented?

Reply:
Apologies, I couldn't find any.  I looked into the code for [stats.Mean](https://github.com/apache/beam/blob/master/sdks/go/pkg/beam/transforms/stats/mean.go#L34) for how this is implemented and tailored this for the katas.

Reply:
Ok noted. Thanks for your info.

Reply:
Yup. Documentation hasn't been updated meaningfully in a long while. This is on the list of things to do before no longer being experimental.